.travis.yml

```diff
@@ -45,7 +45,6 @@ before_cache:
 
 cache:
   directories:
-    - "$HOME/.gradle/caches"
     - "$HOME/.ivy2/cache"
+    - "$HOME/.gradle/caches"
     - "$HOME/.jabba/jdk"
-    - "$HOME/.sbt/boot"
```

Comment:
@dwijnand, can you clarify why this line was added on a previous PR? Is that something that we should or want to add to template control and let it propagate to all samples?

Reply:
Yes

Reply:
I guess you are saying 'yes' to my second question, correct?
> Is that something that we should or want to add to template control and let it propagate to all samples?

Reply:
Yep!  The whole of `~/.sbt` should be cached in fact.

Reply:
I'll follow this up.